setup.cfg

```diff
@@ -148,31 +148,13 @@ ignore_errors=True
 [mypy-pandas.tests.extension.json.test_json]
 ignore_errors=True
 
-[mypy-pandas.tests.indexes.datetimes.test_datetimelike]
-ignore_errors=True
-
-[mypy-pandas.tests.indexes.interval.test_base]
-ignore_errors=True
-
 [mypy-pandas.tests.indexes.interval.test_interval_tree]
 ignore_errors=True
 
-[mypy-pandas.tests.indexes.period.test_period]
-ignore_errors=True
-
 [mypy-pandas.tests.indexes.test_base]
 ignore_errors=True
 
-[mypy-pandas.tests.indexes.test_category]
-ignore_errors=True
-
-[mypy-pandas.tests.indexes.test_numeric]
-ignore_errors=True
-
-[mypy-pandas.tests.indexes.test_range]
-ignore_errors=True
-
-[mypy-pandas.tests.indexes.timedeltas.test_timedelta]
+[mypy-pandas.tests.indexing.test_coercion]
```

Comment:
@srcole thanks for the updates.

looks like merge master went awry at some point. can you remove this.